dataactcore/models/stagingModels.py

```diff
@@ -1073,7 +1073,7 @@ class PublishedAwardProcurement(Base):
     extent_competed = Column(Text)
     extent_compete_description = Column(Text)
     contract_opp_notice = Column(Text)
-    contract_opp_notice_description = Column(Text)
+    contract_opp_notice_desc = Column(Text)
```

Comment:
Dunno if intentional or not. At the very least, it should be consistent with `AwardProcurement`

Reply:
Mistake on my part.